modules/service-bus/namespaces/.test/common/main.test.bicep

```diff
@@ -84,8 +84,8 @@ module testDeployment '../../main.bicep' = {
       trustedServiceAccessEnabled: true
       virtualNetworkRules: [
         {
+          ignoreMissingVnetServiceEndpoint: true
           subnet: {
-            ignoreMissingVnetServiceEndpoint: true
```

Comment:
Hey @lsnoddy, I believe the fix has to be the other way around. The property should be "subnet ResourceId" instead of an object with just one property ID.

Reply:
As such, the template can stay the same 😊

Reply:
It's an approach we implemented usually in cases where an object only has one property